maintainers/maintainer-list.nix

```diff
@@ -7441,6 +7441,12 @@
     githubId = 39707188;
     name = "Lynn Dong";
   };
+  lyrain = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.